core-bundle/src/Resources/contao/classes/Hybrid.php

```diff
@@ -264,7 +264,7 @@ protected function isHidden()
 		}
 
 		// Skip unsaved elements (see #2708)
-		if (!$this->objParent->tstamp)
+		if (isset($this->objParent->tstamp) && !$this->objParent->tstamp)
```

Comment:
```suggestion
		if (!empty($this->objParent->tstamp))
```

Would this work as well?

Reply:
I don't think so, not having a value would be true, but that's not what we want?